yii-taskforce/frontend/views/tasks/index.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/* @var $this yii\web\View */
```

Comment:
Сюда стоит добавить аннотацию передаваемой переменной с заданиями, чтобы PhpStorm подсказывал имена полей
```suggestion
/**
 * @var yii\web\View $this
 * @var Task[] $tasks
 */
```
Ну и необязательный момент, просто на заметку: в phpdoc правильно сначала писать тип, потом имя переменной

Reply:
На всякий случай пишу: я время от времени буду так предлагать изменения, тут можно прям нажать на него и закомитить. Это проще, чем переносить вручную в редактор. Но, впрочем, делай как тебе удобнее.